service/src/main/java/bio/terra/cbas/controllers/RunSetsApiController.java

```diff
@@ -504,6 +506,14 @@ private List<RunStateResponse> buildInputsAndSubmitRun(
         log.warn("Error processing inputs", e);
         runStateResponseList.add(
             storeRun(runId, null, runSet, record.getId(), SYSTEM_ERROR, e.getMessage()));
+      } catch (AzureAccessTokenException e) {
+        log.warn("Unrecognized token", e);
+        runStateResponseList.add(
+            storeRun(runId, null, runSet, record.getId(), SYSTEM_ERROR, e.getMessage()));
+      } catch (DependencyNotAvailableException e) {
+        log.warn("Dependency not available", e);
+        runStateResponseList.add(
+            storeRun(runId, null, runSet, record.getId(), SYSTEM_ERROR, e.getMessage()));
       }
```

Comment:
I had them as 3 different catch blocks since the `log.warn` message was different amongst them all. Would it be cleaner to combine into one catch block and change the message based on which exception is being thrown?